python/utils.py

```diff
@@ -187,19 +188,43 @@ def sweep_ref(I, Rmin, dR, nrow, ncol, dxy, inc, Dx=0., Dy=0., dtype_image='floa
     """
     Compute the intensity map (i.e. the image) given the radial profile I(R)=ints.
     We assume an axisymmetric profile.
+    The output image is assumed with origin in the upper left corner.
 
     Parameters
     ----------
     I: 1D float array
         Intensity radial profile I(R).
-    gridrad: array
-        Radial grid
-    inc: float
-        Inclination, degree
+    Rmin : float
+        Inner edge of the radial grid, i.e. the radius where the brightness is intensity[0].
```

Comment:
Should it be

... i.e. the radius **below which** the brightness is intensity[0].

Reply:
No, for R<Rmin intensity is assumed 0.

Reply:
Ok, then say that instead of `intensity[0]`. 

I just noticed: why do you use brightness and intensity interchangeably? Do they have the same units, is this common practice in radiointerferometry? I would think we should stick with one name

Reply:
I agree I use both of them but they are interchangeable.... Will make this uniform in a future release.
> Using the ray-optics approximation, we can define the specific intensity (sometimes called spectral intensity or spectral brightness, spectral radiance, or loosely, just brightness) I in terms of...

from https://www.cv.nrao.edu/course/astr534/Brightness.html

Reply:
Anyway, in `utils.py` now I use only `intensity`.